frontend/src/components/Logs/LogStateIndicator/LogStateIndicator.styles.scss

```diff
@@ -41,10 +41,4 @@
 			background-color: var(--bg-sakura-500);
 		}
 	}
-
-	&.isActive {
-		.line {
```

Comment:
we do not need the log-state indicator to change the color anymore as we will be giving the log type indicator with some opacity in the background